staging/src/k8s.io/apimachinery/pkg/util/intstr/instr_fuzz.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// setting nofuzz in go build -tags will disable linking in the fuzzer code
+// we do this by default at runtime
+// +build !nofuzz
```

Comment:
I'm used to seeing this RIGHT at the top of the file - I would not think to look for it anywhere else.

Reply:
ack.